generate_release_notes.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+if [ "$1" == "-h" ] || [ "$1" == "--help" ] || [ "$#" -ne 0 ]; then
+  echo "------------------------------------------------------------------------------"
+  echo "Usage - generate_release_notes"
+  echo ""
+  echo "Provide the list of repositories to operate upon in a file named repo_list.txt"
+  echo "-------------------------------------------------------------------------------"
+  exit 0;
+fi;
+
+repoOwner="uc-cdis"
+startDate="2019-12-30"
+endDate="2020-01-24"
+githubAccessToken=<GITHUB PERSONAL ACCESS TOKEN>
+
+if find . -name "release_notes.md" -type f; then
+  echo "Deleting existing release notes"
+  rm -f gen3_release_notes.md
+fi
+
+touch gen3_release_notes.md
+echo "# Core Gen3 Release 202002 (Aurora)" >> gen3_release_notes.md
+echo >> gen3_release_notes.md
+
+repo_list="repo_list.txt"
+while IFS= read -r repo; do
+  echo "### Getting the release notes for repo ${repo} ###"
+  gen3git --repo "${repoOwner}/${repo}" --github-access-token "${githubAccessToken}" --from-date "${startDate}" gen --to-date "${endDate}" --markdown
```

Comment:
Should we do an RC=$? check here in case someone breaks the latest version of the release helper?

Reply:
Added it